Documentation/ceph-object-bucket-claim.md

```diff
@@ -105,13 +105,13 @@ provisioner: rook-ceph.ceph.rook.io/bucket [2]
 parameters: [3]
   objectStoreName: my-store
   objectStoreNamespace: rook-ceph
-  region: us-west-1
-  bucketName: ceph-bucket [4]
-reclaimPolicy: Delete [5]
+  region: us-west-1 [4]
+  bucketName: ceph-bucket [5]
+reclaimPolicy: Delete [6]
 ```
 1. `label`(optional) here associates this `StorageClass` to a specific provisioner.
 1. `provisioner` responsible for handling `OBCs` referencing this `StorageClass`.
-1. **all** `parameter` required.
+1. `region`(optional) defines the region in which the bucket should be created. For RGW, this maps to `zonegroup` of the server. If user does not define `region`, Rook will fill this from in-memory `objectStoreContext` for the `ceph object store` and by default this value is the name of `ceph object store` for the internal cluster. If region is specified by the user and if it is different from default value, then create [CephObjectZoneGroup](ceph-object-multisite-crd.md#object-zone-group-settings) for the `ceph object store`. Please refer to [Ceph's documentation](https://docs.ceph.com/en/latest/radosgw/multisite/#zone-groups) for more details.
```

Comment:
Can we shorten this description? There is a lot of detail the user doesn't need to know, such as "in-memory objectStoreContext". All they need to know is if they want to set a region name, what should their settings be in the CRs? I'm not even clear on how to do it. We might need two separate examples for 1) the simple local object store and 2) the multisite.